### ISSUE TITLE
OPC UA local server multicast support is broken

### ISSUE THREAD

**Issue by kumajaya** [eclipse-4diac/4diac-forte#82]
Without a modified source code, UaExpert from Unified Automation display:
![uaexpert-no-patch](https://github.com/eclipse-4diac/4diac-forte/assets/1124422/229b4735-86ee-4896-9a7d-079174a7f430)

```
diff --git a/src/com/opc_ua/opcua_local_handler.cpp b/src/com/opc_ua/opcua_local_handler.cpp
index d08b9d1..ee8f462 100644
--- a/src/com/opc_ua/opcua_local_handler.cpp
+++ b/src/com/opc_ua/opcua_local_handler.cpp
@@ -165,6 +165,7 @@ void COPC_UA_Local_Handler::configureUAServer(UA_ServerStrings &paServerStrings,
   // hostname will be added by mdns library
   UA_String_clear(&paUaServerConfig.mdnsConfig.mdnsServerName);
   paUaServerConfig.mdnsConfig.mdnsServerName = UA_String_fromChars(paServerStrings.mMdnsServerName.c_str());
+  paUaServerConfig.mdnsEnabled = true;
 #endif //FORTE_COM_OPC_UA_MULTICAST

   UA_String_clear(&paUaServerConfig.customHostname);

```
With above patch applied, FORTE instance run on port 61499 with OPC UA on port 4840 and another FORTE instance run on port 61500 with OPC UA on port 4841, UaExpert display:
![uaexpert-patch](https://github.com/eclipse-4diac/4diac-forte/assets/1124422/8fd4b7b5-baaa-4da6-ba9e-c641f907228d)

With above patch applied, FORTE instance run on port 61500 with OPC UA on port 4841 but without FORTE instance on port 61499 with OPC UA on port 4840, UaExpert display:
![uaexpert-patch-no-4840](https://github.com/eclipse-4diac/4diac-forte/assets/1124422/078c3543-7763-4d83-a696-e83c6629c9bc)
Start official Local Discovery Server service from OPC Foundation will make FORTE to close (crash).

FORTE for win32 target build with Visual Build Tools 2022 - amd64, MSBuild version 17.9.8+b34f75857.

**Comment by kumajaya** [comment]
Proposed fix as a PR: https://github.com/eclipse-4diac/4diac-forte/pull/75 but someone else with a better knowledge to FORTE OPC UA LDS design need to check it.

**Comment by cochicde** [comment]
fixed with #239
